fog/ingest/enclave/impl/tests/tx_processing.rs

```diff
@@ -59,6 +59,7 @@ fn test_ingest_enclave(logger: Logger) {
                 let tx_private_key = RistrettoPrivate::from_random(&mut rng);
                 let e_fog_hint = FogHint::from(&bob_public_address).encrypt(&fog_pubkey, &mut rng);
                 TxOut::new(
+                    BlockVersion::MAX,
```

Comment:
Is it worth for some of these tests to also support previous block versions as well?

Reply:
yeah I guess so